spectral.yaml

```diff
@@ -293,15 +293,6 @@ rules:
     then:
       function: param-order
 
-  az-path-parameter-names:
-    description: Path parameter names should be consistent across all paths.
-    message: '{{error}}'
-    severity: warn
-    formats: ['oas2', 'oas3']
-    given: $.paths
-    then:
-      function: path-param-names
-
```

Comment:
This is an ancillary change that I clubbed into this PR -- it just moves the rule definition down to lines 331-339 to restore alpha ordering of the rules in the file.